### PR TITLE
Bugfix/connection

### PR DESCRIPTION
**Pull request by adonaygyb**

### CODE REVIEW

File: .github/workflows/c-cpp.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+name: C/C++ CI
```
Comment (adonaygyb) [inline .github/workflows/c-cpp.yml:1]: @Injoh great idea setting up this workflow!

File: flask_webapp/src/tester.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
+from device_list import *
```
Comment (adonaygyb) [inline flask_webapp/src/tester.py:1]: I need to remove this file. Not sure how it slipped and got here!

Comment (adonaygyb) [reply flask_webapp/src/tester.py:1]: - [ ] Remove tester.py from pr

File: .github/workflows/c-cpp.yml (hunk 1 above)
Comment (adonaygyb) [reply .github/workflows/c-cpp.yml:1]: Has it been tested? Does it work?

**Review by NerdboyQ — APPROVED** [review]

**Pull request merged**